### ISSUE TITLE
Cache the compilation result

### ISSUE THREAD

**Issue by Yamakaky** [neomura/setup-aseprite-cli-action#9]
Did you find a way to cache the compilation of aseprite? It lasts quite some time and become a bottleneck in my pipeline. Maybe with https://github.com/actions/cache?

**Comment by jameswilddev** [comment]
Oh wow, somebody is using something I made!

I will look into that, thanks!  I hope it's working well for you!

**Comment by Yamakaky** [comment]
Ahah, yes I do, but the build of aseprite lasts 10min ^^

**Comment by jameswilddev** [comment]
Looks like the caching works everywhere but windows-2016, where it doesn't block builds, but doesn't cache, due to what looks like an internal fault in the step.  I'll be releasing 1.0.1 later today which should include this fix.

**Comment by jameswilddev** [comment]
Ah, hit a bit of snag in that composite steps don't support using steps - just running scripts.  Didn't realize.

**Comment by jameswilddev** [comment]
Update on this - we're still waiting for https://github.com/actions/runner/issues/646.

**Comment by jameswilddev** [comment]
Yeah it's quite disappointing.  I'll have to look when I'm a bit less swamped, sorry about that.

**Comment by jameswilddev** [comment]
GitHub have finally enabled this functionality!

**Comment by jameswilddev** [comment]
Unfortunately they haven't yet enabled `if` so we can't skip the build step if the cache is hit.

**Comment by jameswilddev** [comment]
Just running a quick test with the cache check inside a script...

**Comment by jameswilddev** [comment]
@Yamakaky Give v1.0.2 a try!

**Comment by SIRprise** [comment]
I didn't get the problem why you can't upload the cache or artifact?
I did it with a .NET project here [](https://github.com/SIRprise/SSD-LED/blob/master/.github/workflows/dotnetcore.yml) and before GitHub introduced that actions, I used AppVeyor.
